app/src/main/java/de/storchp/opentracks/osmplugin/maps/MapConstants.java

```diff
@@ -0,0 +1,12 @@
+package de.storchp.opentracks.osmplugin.maps;
+
+public class MapConstants {
+
+    /**
+     * @return the default starting zoom level if nothing is encoded in the map file.
+     */
+    public static byte getZoomLevelDefault() {
```

Comment:
If it is only this, wouldn't it be better to just have a static field in the MapsActivity instead of introducing a complete new class?

Reply:
I also agree - I was actually expecting some more things that are related, but somehow it was just this.
I will migrate it back to MapsActivity ;)